src/eventmit.ts

```diff
@@ -1,28 +1,36 @@
 export type EventmitHandler<T> = (value: T) => any;
-export const eventmit = <T>() => {
+
+export type Mitter<T> = {
```

Comment:
```suggestion
export type Eventmitter = {
```

Reply:
We also need to update README.md
https://github.com/azu/eventmit#api (just copy paste)

Reply:
OK, I fixed it.